src/include/uapi/ipc.h

```diff
@@ -763,6 +769,7 @@ struct sof_ipc_comp_eq_fir {
 	struct sof_ipc_comp comp;
 	struct sof_ipc_comp_config config;
 	uint32_t size;
+	uint32_t reserved[16];
 	unsigned char data[0];
```

Comment:
do we want to try an have some sort of alignment, e.g. if you use a uint32_t already then you add a reserved array with 15 values?

Reply:
yes, 32 or 64 bytes are fairly common units, aren't they?